src/savegame_json.cpp

```diff
@@ -225,7 +224,6 @@ void player::store(JsonOut &json) const
     json.member( "body_wetness", body_wetness );
 
     // breathing
-    json.member( "underwater", underwater );
     json.member( "oxygen", oxygen );
```

Comment:
Not sure if I understand

`underwater` is now a creature member. `oxygen` should be just a player one and should be saved just like before.